src/index.html

```diff
@@ -3,11 +3,68 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./styles/index.scss"
+      >
+    <link rel="preconnect"
+      href="https://fonts.googleapis.com"
+      >
```

Comment:
Need to check it and fix everywhere
<img width="1074" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/57f86f97-8099-45d4-83d8-138578b114da">